Sources/TuistGenerator/Utils/EmbedScriptGenerator.swift

```diff
@@ -130,7 +130,7 @@ final class EmbedScriptGenerator: EmbedScriptGenerating {
         echo "mkdir -p ${CONFIGURATION_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}"
         mkdir -p "${CONFIGURATION_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}"
 
-        SWIFT_STDLIB_PATH="${DT_TOOLCHAIN_DIR}/usr/lib/swift/${PLATFORM_NAME}"
+        SWIFT_STDLIB_PATH="${TOOLCHAIN_DIR}/usr/lib/swift/${PLATFORM_NAME}"
```

Comment:
For context, [`DT_TOOLCHAIN_DIR`](https://github.com/CocoaPods/CocoaPods/pull/12009) causes issues with Xcode 15.